manager-bundle/src/Resources/config/autoload.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+/**
+ * @var ContainerBuilder $container
+ */
+return static function(ContainerConfigurator $configurator) use ($container) {
+
+    $originalDefinitions = $container->getDefinitions();
+
+    // Don't do anything if there is a service definition for the App namespace
+    foreach ($originalDefinitions as $id => $definition) {
+        if (0 === strpos($id, 'App\\')) {
+            return;
+        }
+    }
+
+    $config = $configurator->services();
+
+    $config->defaults()->autowire(true);
+    $config->defaults()->autoconfigure(true);
```

Comment:
The `defaults()` method will create a new `DefaultsConfigurator` object upon each call. Maybe we can reuse the returned object?

Reply:
Changed in b14a62f9d73fce75b7a33ca5869c80aeab68ebdc